.circleci/config.yml

```diff
@@ -59,6 +59,19 @@ behat_tests: &behat_tests
         key: terminus_-{{ checksum "composer.lock" }}
         paths:
           - "/tmp/vendor"
+    - restore_cache:
```

Comment:
Heads up that you'll need to rebase or merge master to pick up the changes to master from powr-494-1 before this PR should be approved.